.github/CONTRIBUTING.md

```diff
@@ -238,35 +274,34 @@ This prevents nesting levels from getting deeper then they need to be.
 
 * Calls to the database must be escaped properly - use sanitizeSQL to escape text based database entries from players or admins, and isnum() for number based database entries from players or admins.
 
+* It is recommended as well you do not expose information about the players - even something as simple as the number of people who have readied up at the start of the round can and has been used to try to identify the round type.
```

Comment:
You still probably should change the example to something applicable as you brought up. Keeping information hidden like which player is playing in what role etc.

Also you put this back in the wrong place